### PR TITLE
Adding Contribution guidelines

### LINKED ISSUE #3145: Missing CONTRIBUTING.md file
All K8s subrepositories should have a CONTRIBUTING.md file, which at the minimum should point to https://github.com/kubernetes/community/blob/master/contributors/guide/README.md. Care should be taken that all information is in sync with the contributor guide.

Subrepositories may also have contributing guidelines specific to that repository. They should be explicitly documented and explained in the CONTRIBUTING.md

Ref:  https://github.com/kubernetes/community/issues/1832

### PR DESCRIPTION
**Pull request by rohanand**
Adding Contribution guidelines for dashboard.
fixes #3145

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Contributing Guidelines
+The Kubernetes Dashboard Project is a web-based UI for Kubernetes clusters.It allows users to manage the cluster as well as application running in the cluster.All the contribution is done via github.Please also read the contributors guide
```
Comment (jeefy) [inline CONTRIBUTING.md]: Space needed between "clusters." and "It"

Comment (jeefy) [reply CONTRIBUTING.md]: Space needed between "cluster." and "All"

Comment (jeefy) [reply CONTRIBUTING.md]: Capitalize G and H in GitHub

Comment (jeefy) [reply CONTRIBUTING.md]: Space needed between "Github." and "Please"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
+
+### Sign the CLA
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests. Please see https://github.com/kubernetes/community/blob/master/CLA.md for more info
+
+### Learn a bit about the kubernetes dashboard
```
Comment (jeefy) [inline CONTRIBUTING.md]: Capitalize Kubernetes and Dashboard

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Before managing and running application in kubernetes dashboard it is recommended to go through the doc
```
Comment (jeefy) [inline CONTRIBUTING.md]: Capitalize Kubernetes and Dashboard.

Maybe rephrase? "To learn more about the Kubernetes Dashboard project, please read through the user documentation:"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+### Sign the CLA
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests. Please see https://github.com/kubernetes/community/blob/master/CLA.md for more info
+
+### Learn a bit about the kubernetes dashboard
+Before managing and running application in kubernetes dashboard it is recommended to go through the doc 
+https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/
+
+### Contributing A Patch
+
+1. Submit an issue by describing  your proposed change to the repo in question.
```
Comment (jeefy) [inline CONTRIBUTING.md]: Remove a space between "describing" and "your"

**Comment by maciaszczykm** [comment]
https://github.com/kubernetes/dashboard/issues/3145#issuecomment-405604476

**Comment by rohanand** [comment]
@jeefy I have updated CONTRIBUTING.md as per your comment.Please review.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Contributing Guidelines
+The Kubernetes Dashboard Project is a web-based UI for Kubernetes clusters. It allows users to manage the cluster as well as application running in the cluster. All the contribution is done via GitHub. Please also read the contributors guide:
```
Comment (jeefy) [inline CONTRIBUTING.md]: application -> applications

Comment (rohanand) [reply CONTRIBUTING.md]: @jeefy I have updated the changes.

**Comment by jeefy** [comment]
/lgtm

We may want to clean / remove the old CONTRIBUTING.md from the .github/ folder in the future.

**Comment by konryd** [comment]
Thanks @rohanand and thanks @jeefy for reviewing!
